src/main/java/com/onlinebookstore/service/impl/BookServiceImpl.java

```diff
@@ -36,4 +36,18 @@ public List<BookDto> findAll() {
                 .map(bookMapper::toDto)
                 .toList();
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public BookDto updateById(Long id, CreateBookRequestDto requestDto) {
+        bookRepository.findById(id);
+        Book updatedBook = bookMapper.toModel(requestDto);
+        updatedBook.setId(id);
```

Comment:
```suggestion
        bookMapper.updateBookFromDto(bookDto, book);
```

Reply:
> 
I tried using bookDto, but it just doesn't work because I use the CreateBookRequestDto method to save and update the book. Is it possible to leave it like that or is it better to change the logic?